scripts/release_scripts/initial_release_prep.py

```diff
@@ -0,0 +1,321 @@
+
+    return list(set(jobs_to_run))
+
+
+def did_supported_audio_languages_change(
+        remote_alias, previous_release_version):
+    """Checks changes in constants.SUPPORTED_AUDIO_LANGUAGES between
+    the current and previous release.
+
+    Args:
+        remote_alias: str. The alias for Oppia repo.
+        previous_release_version: str. The version of the previous release.
+
+    Returns:
+        bool. Whether supported audio languages have changed.
+    """
+    try:
+        from constants import constants
+        supported_audio_languages_for_current_release = constants[
+            'SUPPORTED_AUDIO_LANGUAGES']
+
+        common.run_cmd([
+            'git', 'checkout',
+            '%s/release-%s' % (remote_alias, previous_release_version),
+            '--', 'assets/constants.ts'])
+        from constants import constants
+        supported_audio_languages_for_previous_release = constants[
+            'SUPPORTED_AUDIO_LANGUAGES']
+    finally:
+        common.run_cmd(['git', 'reset', 'assets/constants.ts'])
+        common.run_cmd(['git', 'checkout', '--', 'assets/constants.ts'])
+
+    return (
+        sorted(supported_audio_languages_for_current_release) != sorted(
+            supported_audio_languages_for_previous_release))
+
+
+def main():
+    """Performs task to initiate the release."""
+    common.require_cwd_to_be_oppia()
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.RELEASE_NOTES_URL)
+    common.ask_user_to_confirm(
+        'Please check if anything extra is required for the release. '
+        'If so, keep track of this and do it at the appropriate point.')
+
+    common.open_new_tab_in_browser_if_possible(
+        release_constants.RELEASE_DRIVE_URL)
+    python_utils.PRINT(
+        'Has the QA lead created a document for the current release?\n'
+        'Confirm by entering y/ye/yes.\n')
+    doc_for_release_created = python_utils.INPUT().lower()
+    if doc_for_release_created not in (
+            release_constants.AFFIRMATIVE_CONFIRMATIONS):
+        raise Exception(
+            'Please ensure a new doc is created for the '
+            'release before starting with the release process.')
+
+    remote_alias = common.get_remote_alias(release_constants.REMOTE_URL)
+    python_utils.PRINT('Enter version of previous release.')
+    previous_release_version = python_utils.INPUT()
+    assert re.match(r'^\d+\.\d+\.\d+$', previous_release_version)
+
+    extra_jobs_to_run = get_extra_jobs_due_to_schema_changes(
+        remote_alias, previous_release_version)
+    if did_supported_audio_languages_change(
+            remote_alias, previous_release_version):
+        # This job is run so that an opportunity is created for
+        # contributors to translate and voiceover an exploration
+        # whenever a new audio language is added.
+        # Refer: https://github.com/oppia/oppia/issues/8027.
+        extra_jobs_to_run.append(
```

Comment:
We want to run this job only if a new language is added or we delete a language, but `did_supported_audio_languages_change` will trigger this on any changes in the languages_ list. Should we show the changes and let the user decided whether we need to run the job? 

(This will may create confusion and a complicated workflow, I'm fine with the current workflow but wanted to know whether it's fine to run the job sometime (Which I don't think will happen frequently) when it's not required in cost of having a simple and clear workflow. @seanlip any suggestion?)

Reply:
<Ignore this comment, I've suggested a better way to handel this in the comment below.>